vmm.mk

```
@@ -75,15 +76,16 @@ libvmm/arch/aarch64/vgic:
 libvmm.a: ${OBJECTS}
 	ar rv $@ $^
 
+${OBJECTS}: ${SDDF}/include
 ${OBJECTS}: ${CHECK_LIBVMM_CFLAGS} |libvmm/arch/aarch64/vgic
```

```suggestion
${OBJECTS}: ${CHECK_LIBVMM_CFLAGS}
```

should we remove `|libvmm/arch/aarch64/vgic`?

---

No, it's needed to create the directory that the objects will be put into.